tests/common/sequester.py

```diff
@@ -15,35 +19,27 @@
     StorageSequesterService,
     WebhookSequesterService,
 )
-from parsec.crypto import SigningKey
-from parsec.sequester_crypto import (
-    SequesterEncryptionKeyDer,
-    SequesterVerifyKeyDer,
-    sequester_authority_sign,
-)
 
 
 @dataclass
 class SequesterAuthorityFullData:
     certif: bytes
     certif_data: SequesterAuthorityCertificate
-    signing_key: oscrypto.asymmetric.PrivateKey
-    verify_key: oscrypto.asymmetric.PublicKey
+    signing_key: SequesterSigningKeyDer
+    verify_key: SequesterVerifyKeyDer
 
 
 def sequester_authority_factory(
-    organization_root_signing_key: SigningKey, timestamp: Optional[DateTime] = None
+    organization_root_signing_key: SigningKey, timestamp: DateTime | None = None
 ) -> SequesterAuthorityFullData:
     timestamp = timestamp or DateTime.now()
-    # Don't use such a small key size in real world, this is only for test !
-    # (RSA key generation gets ~10x slower between 1024 and 4096)
-    verify_key, signing_key = oscrypto.asymmetric.generate_pair("rsa", bit_size=1024)
+    priv_key = SequesterPrivateKeyDer.generate()
```

Comment:
So what the size of the RSA key are using here ? no need for the 1024bits key size trick anymore ?

Reply:
4096, 1024 bits are used only in rust test

Reply:
Done